AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -20,4 +22,25 @@
  */
 - (void)recursivelyEnsureDisplaySynchronously:(BOOL)synchronously;
 
+- (void)drawRect:(CGRect)bounds withParameters:(id <NSObject>)parameters isCancelled:(asdisplaynode_iscancelled_block_t)isCancelledBlock isRasterizing:(BOOL)isRasterizing;
+
+- (UIImage *)displayWithParameters:(id <NSObject>)parameters isCancelled:(asdisplaynode_iscancelled_block_t)isCancelled;
+
+/**
+ * @abstract allow modification of a context before the node's content is drawn
+ *
+ * @discussion Set the block to be called after the context has been created and before the node's content is drawn.
+ * You can override this to modify the context before the content is drawn. You are responsible for saving and
+ * restoring context if necessary. Restoring can be done in contextDidDisplayNodeContent
+ * This block can be called from *any* thread and it is unsafe to access any UIKit main thread properties from it.
+ */
+@property (nonatomic, strong) ASDisplayNodeContextModifier willDisplayNodeContentBlock;
```

Comment:
I actually don't have a better name suggestion, but wonder if there is something we can do to improve this one. In particular, the name does not particularly capture that this Block is called from inside the drawing operation itself.